src/com/pentalog/pentastagiu/homework2/bank/Account.java

```diff
@@ -0,0 +1,36 @@
+package com.pentalog.pentastagiu.homework2.bank;
+
+public class Account {
+
+    private  Integer balance;
+
+    public Account() {
+        this.balance = 0;
+    }
+
+    public  Integer getBalance() {
+        return this.balance;
+    }
+
+    public  void setBalance(Integer balance) {
```

Comment:
ti-as sugera cand implementezi o clasa sa incepi cu fielduri private si sa generezi getter/setteri(mai ales) pe masura ce ai nevoie de ei; este nevoie de metoda setBalance? e suficient sa avem cele 2 metode de withdraw si deposit in care aplicam toate validarile de care este nevoie si evitam astfel sa ii dam control cuiva sa seteze balance cu ce valoare vrea